.travis.yml

```diff
@@ -36,6 +34,6 @@ script:
       fi
 
     - if [[ $TEST_TARGET == 'coding_standards' ]]; then
-        pycodestyle --ignore=E741,E226 --max-line-length=100 tests ;
-        pycodestyle --ignore=E741,E226 --max-line-length=100 utide ;
+        pycodestyle --ignore=E741,E226,W504 --max-line-length=100 tests ;
```

Comment:
`pycodestyle`, by default, ignores:
E121, E123, E126, E133, E226, E241, E242, E704, W503, W504

I think this is true, unless you specify the `ignore` flag. Is there any others from that list that we want to include?

Reply:
I don't think so. To be honest some of those I would like to remove from that list, like W504. But that is low priority and I don't think we should worry about it now.